pontoon/base/utils.py

```diff
@@ -681,14 +681,14 @@ def glob_to_regex(glob):
     regex = r""
     for part in pattern:
         if isinstance(part, Starstar):
-            regex += r"([\w/]*)"
+            regex += r"([^/]*)"
         elif isinstance(part, Star):
-            regex += r"([\w]*)"
+            regex += r"(.*)"
```

Comment:
Oops, sorry, this is vice-versa. `**` is everything, `*` is only single-hiearchy wildcard, aka, non-`/`.

Reply:
Meh, sorry, `**` is actually slightly different, looking back at the c-l source.

I wonder, maybe the easier way to get rid of the named groups from c-l, via `re.sub(r'/^\(?P<s[0-9]+>', '')` , and then strip the last char, too?